applications/sso/scripts/startup/wait-for-keycloak.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+waitingForKeycloakEndpoint=true
+
+while $waitingForKeycloakEndpoint; do
+  if [[ $(curl --head --write-out %{http_code} --silent --output /dev/null http://localhost:8084/auth) != 303 ]]
```

Comment:
vi har et kul hjelpeskript, wait_for_http_200, som var tatt fra internett. kan vi ikke gjenbruke det?

Reply:
Måtte endre på det hvis jeg skulle bruke det. Får 303, ikke 200, og må få det til å kjøre i bakgrunnen, 'as is' vil det stoppe hele startup-prosessen.